### PR TITLE
feat: updated readme

### PR DESCRIPTION
**Pull request by samuelOsborne**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,21 +12,29 @@
 
 This is the Rust implementation of the dotLottie player and its related tools. It utilizes uniffi-rs to generate FFI bindings for Kotlin, Swift, and WebAssembly (WASM). these bindings are then used in the native dotLottie players for [Android](https://github.com/LottieFiles/dotlottie-android), [iOS](https://github.com/LottieFiles/dotlottie-ios), and [Web](https://github.com/LottieFiles/dotlottie-web) bringing consistency of playback and dotLottie features across all platforms.
```
Comment (theashraf) [inline README.md:13]: Should we include:

- The problem dotlottie-rs solves (fragmentation and rendering inconsistency issues)
- The features it supports

File: README.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # dotLottie Rust
 
-![CI](https://github.com/LottieFiles/dotlottie-rs/workflows/release/badge.svg)
+![CI](https://img.shields.io/github/v/release/LottieFiles/dotlottie-rs)
```
Comment (theashraf) [inline README.md:3]: nice!!

File: README.md [hunk 3]
```diff
@@ -90,6 +98,12 @@ The release PR should be checked for correctness and then merged. Once that is d
 Github Actions workflow will be started automatically to do the work of actually creating the new
 release and building & uploading the related release artifacts.
 
+### Relevant Tools
+
+- [For your dotLottie creation and modification needs](https://github.com/dotlottie/dotlottie-js)
+- [Tools for parsing Lottie animations](https://github.com/LottieFiles/relottie)
+- [dotLottie playground for creating multi-animations dotLotties](https://dotlottie.github.io/player-component/)
```
[the remark this answers is not in the document]
Comment (samuelOsborne) [reply README.md]: have removed, can add it back when we complete the repo move

File: README.md (hunk 1 above)
Comment (samuelOsborne) [reply README.md:13]: added @theashraf please review

**Review by theashraf — APPROVED** [review]

**Pull request merged**